Data/Migrations/000014_user_file_improvement.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS `user_file_new`
+(
+    `Id`        INTEGER PRIMARY KEY AUTOINCREMENT  NOT NULL,
+    `OwnerId`   INTEGER                            NOT NULL,
+    `Path`      VARCHAR(2147483647) COLLATE NOCASE NOT NULL,
+    `Type`      INTEGER                            NOT NULL,
+    `CreatedAt` TEXT                               NOT NULL,
+    `UpdatedAt` TEXT                               NOT NULL
+);
+
+INSERT INTO user_file_new
+SELECT Id,
+       OwnerId,
+       Path,
+       Type,
+       CreatedAt,
+       CreatedAt
+FROM user_file;
+
+DROP TABLE user_file;
+ALTER TABLE user_file_new
+    RENAME TO user_file;
+
+DELETE FROM user_file
+WHERE Id IN (
+    SELECT Id
+    FROM (
+        SELECT Id, 
+               ROW_NUMBER() OVER (PARTITION BY OwnerId, Type ORDER BY CreatedAt ASC) AS rn
+
+        FROM user_file
```

Comment:
Filter by type is required. Without it, we will remove all replays/screenshots from DB except one for users.

```suggestion
        FROM user_file
        WHERE Type IN (1, 2) -- Avatar and Banner
```